### PR TITLE
Login avatar btn

### PR DESCRIPTION
**Pull request by jammyk**
* Creation of auth component (wrapper component for protected routes)
* Creation of helper classes such as authentication related to cookie setting/removal of user
* Creation of colors constant file

* Creation of user card display portfolio data and actions.

### CODE REVIEW

File: artofinvesting-frontend/src/components/Registration.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    handleSubmit(event) {
+        const {
+            email,
+            password,
+            password_confirmation
+        } = this.state;
+        console.log("form submitted");
+        // axios.post(http://localhost:3001/registration, {
```
Comment (ktory1) [inline artofinvesting-frontend/src/components/Registration.js:33]: 나중에 쓸려고 comment out 해놓은거야?

File: artofinvesting-frontend/src/components/authentication/Authenticated.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    componentDidMount() {
+        getUser(this);
+
+        if (isAuthenticated) {
+          this.setState({ user: { loggedIn: true}});
+        } else {
+          redirectToLogin(this);
+        }
+
+        // const jwt = getJwt();
```
Comment (ktory1) [inline artofinvesting-frontend/src/components/authentication/Authenticated.js:25]: 이것도 필요없음 지워도 될듯

File: artofinvesting-frontend/src/components/authentication/LoginForm.js [hunk 3]
```diff
@@ -1,13 +1,49 @@
...
+        };
+
+        this.onFinish = this.onFinish.bind(this);
+    }
 
     onFinish(values) {
         console.log('Received values of form: ', values);
-    }
 
+        // axios.post(http://localhost:3001/registration, {
```
Comment (ktory1) [inline artofinvesting-frontend/src/components/authentication/LoginForm.js:23]: 이것도 ㅋㅋㅋ

File: artofinvesting-frontend/src/components/Registration.js (hunk 1 above)
Comment (jammyk) [reply artofinvesting-frontend/src/components/Registration.js:33]: 네네 일단은 axios을 안넣어놔서 일단 포맷만 잡아놨어요

**Pull request merged**